sources/api/schnauzer/src/helpers.rs

```diff
@@ -1170,6 +1185,179 @@ pub fn etc_hosts_entries(
     Ok(())
 }
 
+/// This helper validates user input for imageGCHighThresholdPercent and
+/// imageGCLowThresholdPercent, then `trace`s an error message if the input
+/// is out of bounds. Otherwise, this helper accepts the setting.
+/// ! NOTE: the parameters MUST be specified as imageGCHighThresholdPercent first,
+/// then imageGCLowThresholdPercent second.
+pub fn image_gc_threshold_percent(
+    helper: &Helper<'_, '_>,
+    _: &Handlebars,
+    _: &Context,
+    renderctx: &mut RenderContext<'_, '_>,
+    out: &mut dyn Output,
+) -> Result<(), RenderError> {
+    // To give context to our errors, get the template name (e.g. what file we are rendering), if available.
+    trace!("Starting image_gc_threshold_percent helper");
+    let template_name = template_name(renderctx);
+    trace!("Template name: {}", &template_name);
+
+    // Check number of parameters, must be exactly two (high and low GC threshold percentages)
+    trace!("Number of params: {}", helper.params().len());
+    check_param_count(helper, template_name, 2)?;
+
+    // ----- Get the GC Threshold Percentage values out of the template.
```

Comment:
Is this meant to be a "start of section" comment? It's probably not worth trying to convey that with a few leading `-`s.